src/otel.rs

```diff
@@ -24,9 +24,7 @@ pub struct OtelHeaderExtractor<'a> {
 
 impl<'a> Extractor for OtelHeaderExtractor<'a> {
     fn get(&self, key: &str) -> Option<&str> {
-        self.inner
-            .get(key)
-            .and_then(|s| s.iter().next().map(|s| s.as_str()))
+        self.inner.get(key).map(|s| s.as_str())
```

Comment:
Dang, this reminded me we still haven't published the shared wasmCloud crates so we could delete this file :/